### ISSUE TITLE
Korean illustration captions use character that is not available in sans font: "그림 14:#LONGHORN UI 개요"

### ISSUE THREAD

**Issue by ghost** [doc-suse-com/l10n#9]
Korean version of DC-RKE2-SAP-DI31:

![rke2-di31](https://user-images.githubusercontent.com/5476547/130670125-1e3b6f21-8cde-4ce8-8600-5444aab4ce02.png)

**Comment by jfaltenbacher** [comment]
Hi Stefan, is there anything the localization team or Vistatec can do here?

**Comment by ghost** [comment]
Nope, our issue. Though I am not quite sure where the # even comes from, there is no character after the `:`. (It's definitely also not caused by the document--because that does not have extraneous characters either.)

https://github.com/openSUSE/suse-xsl/blob/6e7f886b158da4e2787cca2262a8e41276ba92db/suse2021-ns/common/l10n/ko.xml#L62

**Comment by ptinley-vt** [comment]
It is a non-breaking space character, which I think is added in the creation of the label; for all labels, Figure, Table, Procedure, etc. The KO font being used seemingly does not support the no-break-space character.
